packages/common/src/store/pages/chat/selectors.ts

```diff
@@ -172,7 +172,7 @@ export const getSingleOtherChatUser = (
  * Gets a list of the users the current user has chats with.
  * Note that this only takes the first user of each chat that doesn't match the current one,
  * so this will need to be adjusted when we do group chats.
- */
+ **/
```

Comment:
Not but I think single star is standard?

Reply:
Oh yeah you're right. It was super weird I was getting an 'unterminated comment' error and this fixed it, but I can try again without.